shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -635,87 +642,108 @@ genesisId ::
   (Crypto crypto) =>
   TxId crypto
 genesisId =
-  TxId $
-    hash
-      ( TxBody
-          Set.empty
-          StrictSeq.Empty
-          StrictSeq.Empty
-          (Wdrl Map.empty)
-          (Coin 0)
-          (SlotNo 0)
-          SNothing
-          SNothing
-      )
-
--- | Creates the UTxO for a new ledger with the specified transaction outputs.
-genesisCoins ::
-  (Crypto crypto) =>
-  [TxOut crypto] ->
-  UTxO crypto
-genesisCoins outs =
-  UTxO $
-    Map.fromList [(TxIn genesisId idx, out) | (idx, out) <- zip [0 ..] outs]
-
--- | Creates the ledger state for an empty ledger which
---  contains the specified transaction outputs.
+  TxId $ hash
+    ( TxBody
+       Set.empty
+       StrictSeq.Empty
+       StrictSeq.Empty
+       (Value Map.empty)
+       (Wdrl Map.empty)
+       (Coin 0)
+       (SlotNo 0)
+       SNothing
+       SNothing
+    )
+
+-- |Creates the UTxO for a new ledger with the specified transaction outputs.
+genesisCoins
+  :: (Crypto crypto)
+  => [TxOut crypto]
+  -> UTxO crypto
+genesisCoins outs = UTxO $
+  Map.fromList [(TxIn genesisId idx, UTxOOut (getAddressTx out) (valueToCompactValue $ getValueTx out)) |
+    (idx, out) <- zip [0..] outs]
+
+-- |Creates the ledger state for an empty ledger which
+-- contains the specified transaction outputs.
 genesisState ::
   Map
     (KeyHash 'Genesis crypto)
     (KeyHash 'GenesisDelegate crypto, Hash crypto (VerKeyVRF crypto)) ->
   UTxO crypto ->
   LedgerState crypto
-genesisState genDelegs0 utxo0 =
-  LedgerState
-    ( UTxOState
-        utxo0
-        (Coin 0)
-        (Coin 0)
-        emptyPPPUpdates
-    )
-    (DPState dState emptyPState)
+genesisState genDelegs0 utxo0 = LedgerState
+  (UTxOState
+    utxo0
+    (Coin 0)
+    (Coin 0)
+    emptyPPPUpdates)
+  (DPState dState emptyPState)
   where
     dState = emptyDState {_genDelegs = GenDelegs genDelegs0}
 
--- | Implementation of abstract transaction size
-txsize :: forall crypto. (Crypto crypto) => Tx crypto -> Integer
-txsize tx = numInputs * inputSize + numOutputs * outputSize + rest
+-- get total size of outputs in a Tx
```

Comment:
@JaredCorduan 
`allOutputsSize` is used to calculate the size out the outputs of a tx to be used in the txsize /minfee calculation
`scaledSizeCompactValue` is used in the STS/Utxo.hs transition file to compute the size of the deposit needed for the output